internal/rootcoord/root_coord.go

```diff
@@ -2901,6 +2900,22 @@ func (c *Core) ListPolicy(ctx context.Context, in *internalpb.ListPolicyRequest)
 			Status: merr.StatusWithErrorCode(errors.New(errMsg), commonpb.ErrorCode_ListPolicyFailure),
 		}, nil
 	}
+	// expand privilege groups and turn to policies
+	allGroups, err := c.getPrivilegeGroups(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
According to the code above, the error is supposed to be returned by a wrapped response. Is it safe to return a nil response here?

Reply:
fixed